src/Elliptic/Systems/Punctures/BoundaryConditions/Flatness.hpp

```diff
@@ -0,0 +1,77 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <cstddef>
+#include <string>
+#include <vector>
+
+#include "DataStructures/Tensor/TypeAliases.hpp"
+#include "Domain/Tags.hpp"
+#include "Elliptic/BoundaryConditions/BoundaryCondition.hpp"
+#include "Elliptic/BoundaryConditions/BoundaryConditionType.hpp"
+#include "Options/Options.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+class DataVector;
+/// \endcond
+
+namespace Punctures::BoundaryConditions {
+
+/// Impose asymptotic flatness boundary conditions $\partial_r(ru)=0$
+class Flatness : public elliptic::BoundaryConditions::BoundaryCondition<3> {
+ private:
+  using Base = elliptic::BoundaryConditions::BoundaryCondition<3>;
+
+ public:
+  static constexpr Options::String help = "Asymptotic flatness d_r(ru)=0";
+  using options = tmpl::list<>;
+
+  Flatness() = default;
+  Flatness(const Flatness&) = default;
+  Flatness& operator=(const Flatness&) = default;
+  Flatness(Flatness&&) = default;
+  Flatness& operator=(Flatness&&) = default;
+  ~Flatness() = default;
+
+  /// \cond
+  explicit Flatness(CkMigrateMessage* m) : Base(m) {}
+  using PUP::able::register_constructor;
+  WRAPPED_PUPable_decl_template(Flatness);
+  /// \endcond
+
+  std::unique_ptr<domain::BoundaryConditions::BoundaryCondition> get_clone()
+      const override {
+    return std::make_unique<Flatness>(*this);
+  }
+
+  std::vector<elliptic::BoundaryConditionType> boundary_condition_types()
+      const override {
+    return {1, elliptic::BoundaryConditionType::Neumann};
```

Comment:
either switch to `()` or `{elliptic::BoundaryConditionType::Neumann}`. The current is a bit tough to parse with `std::vector`s annoying constructors. I think one (or both) of the suggestions should work and are unambiguous/avoid the silliness that is std::vector construction